test/Utilities/TestFramework.ForTestingMSTest/TestFramework.ForTestingMSTest.csproj

```diff
@@ -1,20 +1,16 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netstandard2.0;$(MicrosoftTestingTargetFrameworks)</TargetFrameworks>
```

Comment:
Why is `netstandard2.0` not enough?

Reply:
It was for polyfill but I think I got rid of the deps there so I would need to check if that's enough with netstandard

Reply:
Keeping it so that we can benefit from platform optimizations